src/main/java/uk/gov/pay/api/model/Charge.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.pay.api.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Charge {
+
+    @JsonProperty("charge_id")
+    private String chargeId;
+    private Long amount;
+    private String status;
+    @JsonProperty("return_url")
+    private String returnUrl;
+    private String description;
+    private String reference;
+    @JsonProperty("payment_provider")
+    private String paymentProvider;
+    private String created_date;
+    @JsonProperty("links")
+    private List<ChargeLink> links = new ArrayList<>();
+
+    public String getChargeId() {
+        return chargeId;
+    }
+
+    public Long getAmount() {
+        return amount;
+    }
+
+    public String getStatus() {
+        return status;
+    }
+
+    public String getReturnUrl() {
+        return returnUrl;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public String getReference() {
+        return reference;
+    }
+
+    public String getPaymentProvider() {
+        return paymentProvider;
+    }
+
+    public String getCreated_date() {
+        return created_date;
+    }
+
+    public List<ChargeLink> getLinks() {
+        return links;
+    }
+}
```

Comment:
Connector return more things, what about ChargeConnectorResponse? what do you suggest?